packages/webview/src/styles/comments.css

```
@@ -544,3 +544,293 @@
   margin-right: 8px;
 }
 
+/* ===== Dark Theme ===== */
+body[data-vscode-theme-kind="vscode-dark"] .comment-modal,
+body[data-vscode-theme-kind="vscode-high-contrast"] .comment-modal {
+  background: rgba(30, 30, 46, 0.95);
```

Stylelint (17.2.0)

[error] Expected "0.95" to be "95%" (alpha-value-notation)

(alpha-value-notation)